### PR TITLE
Git-Workshop/new commit

### PR DESCRIPTION
**Pull request by tnamdevnote**
New pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +79,13 @@ Now you may work on the codebase directly:
...
+You may also contribute without being added as a member of the collaborative team. In this case, you'll submit pull requests using a forked copy of the repo. In short:
 
 - You'll create a copy of the repo and save it to your GitHub account.
 - This forked copy can be synched with the original, so you can be sure to have the latest updates.
 - You may submit pull requests targeting the default `staging` branch.
 
-GitHub has some [excellent documentation](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project) for how to fork a public repo and submit a pull request.  When creating the forked copy, the `staging` branch will automatically be chosen as the default branch.
+GitHub has some [excellent documentation](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project) for how to fork a public repo and submit a pull request. When creating the forked copy, the `staging` branch will automatically be chosen as the default branch.
+
+### Something changed
```
Comment (slothwerks-studio) [inline README.md]: @tnamdevnote Go ahead and remove line 91, please and thank you.  😸

File: README.md [hunk 2]
```diff
@@ -1,59 +1,71 @@
...
 
 ### Date, time, and location
+
 - Monday, June 24th 6pm - 8pm EST
 - Start Garden, Grand Rapids, Michigan ([map](https://www.google.com/maps/place/Start+Garden/@42.9661537,-85.6699129,15z/data=!4m6!3m5!1s0x8819adc447a3b737:0x7a6a91c9a38e58fc!8m2!3d42.9661537!4d-85.6699129!16s%2Fg%2F1ptyfjd85?entry=ttu))
 
 ### RSVP
-Tickets for this event are __FREE__ but limited.  Please RSVP online:
+
+Tickets for this event are **FREE** but limited. Please RSVP online:
```
Comment (slothwerks-studio) [inline README.md:16]: 👍

File: README.md [hunk 3]
```diff
@@ -1,59 +1,71 @@
 # Git Workshop
+
 ## Brought to you by [GRWebDev](https://www.meetup.com/grwebdev/) Powered by [LaFleur Marketing](https://lafleur.marketing/)
 
 ### Welcome!
-At this workshop you'll learn the basics of [Git](https://git-scm.com/), a version control system.  This technology is the basis for numerous popular platforms, including [GitHub](https://github.com/).
+
+At this workshop you'll learn the basics of [Git](https://git-scm.com/), a version control system. This technology is the basis for numerous popular platforms, including [GitHub](https://github.com/).
```
Comment (slothwerks-studio) [inline README.md:7]: Sooooooo... folks don't appreciate my old school "two spaces after a period or colon"?  Come on; the web automatically gets rid of these anyway, right?  😆

[https://www.grammarly.com/blog/spaces-after-period/](https://www.grammarly.com/blog/spaces-after-period/)

I mean, check out those examples.  Don't they look better with two spaces?  😉

**Review by slothwerks-studio — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (slothwerks-studio) [reply README.md:7]: @tnamdevnote (I'm being silly of course.  A single space after a period is fine.  👍)

Comment (tnamdevnote) [reply README.md:7]: Wait, I didn't know that two spaces after a period was a proper punctuation. It's no surprise that two spaces reads better than single spaces in other docs. I guess prettier isn’t very respectful of traditions like kids these days (definitely not me). 🤣

File: README.md (hunk 1 above)
Comment (tnamdevnote) [reply README.md]: Removed!

File: README.md (hunk 3 above)
Comment (slothwerks-studio) [reply README.md:7]: @tnamdevnote Well, two spaces was proper back... ummmmm... many decades ago.  😅  Per Grammarly, this isn't a thing anymore, and the web doesn't render multiple spaces without actual non-breaking spaces (`&nbsp;`).  But decades of habits are hard to break, and I get yelled at all the time for my double-spacing habits.  😉

Comment (slothwerks-studio) [reply README.md:7]: @tnamdevnote Check out this Reddit post, and then all of the comments.  HILARIOUS.  😆

[https://www.reddit.com/r/GenX/comments/tbf0ai/fellow_genxers_stop_double_spacing_after_periods/](https://www.reddit.com/r/GenX/comments/tbf0ai/fellow_genxers_stop_double_spacing_after_periods/)

**Review by slothwerks-studio — APPROVED** [review]

**Comment by slothwerks-studio** [comment]
@tnamdevnote Merge merge merge!  🚀🚀🚀

File: README.md (hunk 3 above)
Comment (tnamdevnote) [reply README.md:7]: My favorite one. 😂 I cannot imagine a world without Oxford comma... 

image

**Pull request merged**